package.json

```diff
@@ -93,6 +93,7 @@
     "react-measure": "^2.0.0",
     "react-nouislider": "^2.0.0",
     "react-onclickoutside": "^6.4.0",
+    "react-portal": "^4.1.5",
```

Comment:
@jherey What's this package for?

Reply:
@fortunee I had issues setting up the project, so I installed it so as to render the DOM but forgot to uninstall when I figured it out. I'll remove and make a new PR.

Reply:
You don't have to make a new PR. Just remove and push @jherey

Reply:
Ok, I'll do that.

Reply:
Done @fortunee